README.md

```diff
@@ -132,24 +132,24 @@ SIG-specific GitHub discussions.
-| <a id="sig-prometheus"></a>Prometheus Interoperability | Every other Wednesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8) | [#otel-prometheus](https://cloud-native.slack.com/archives/C01LSCJBXDZ) | [calendar-prometheus](https://groups.google.com/a/opentelemetry.io/g/calendar-prometheus) |  | [Alolita Sharma](https://github.com/alolita) | 
-| <a id="sig-faas"></a>Functions as a Service (FAAS) | Every other Wednesday at 8:00 PT | [Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc) | [#otel-faas](https://cloud-native.slack.com/archives/C04HVBETC9Z) | [calendar-faas](https://groups.google.com/a/opentelemetry.io/g/calendar-faas) |  | [Austin Parker](https://github.com/austinlparker) | 
-| <a id="sig-profiles"></a>Profiling | Every other Thursday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0) | [#otel-profiles](https://cloud-native.slack.com/archives/C03J794L0BV) | [calendar-profiling](https://groups.google.com/a/opentelemetry.io/g/calendar-profiling) | [Josh Suereth](https://github.com/jsuereth),<br/>[Tigran Najaryan](https://github.com/tigrannajaryan) | [Morgan McLean](https://github.com/mtwo) | 
-| <a id="sig-mainframes"></a>OpenTelemetry on Mainframes | Tuesday at 10:00 PT | [Google Doc](https://docs.google.com/document/d/14p-bpofozTL4n3jy6HZH_TKjoOXvog18G1HBRqq6liE) | [#otel-mainframes](https://cloud-native.slack.com/archives/C05PXDFTCPJ) | [calendar-mainframe](https://groups.google.com/a/opentelemetry.io/g/calendar-mainframe) | [Alolita Sharma](https://github.com/alolita),<br/>[Daniel Dyla](https://github.com/dyladan),<br/>[Morgan McLean](https://github.com/mtwo) | [Morgan McLean](https://github.com/mtwo) | 
-| <a id="sig-client-side-telemetry"></a>Client Instrumentation | Tuesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w) | [#otel-client-side-telemetry](https://cloud-native.slack.com/archives/C0239SYARD2) | [calendar-client-side](https://groups.google.com/a/opentelemetry.io/g/calendar-client-side) |  | [Daniel Gomez Blanco](https://github.com/danielgblanco) | 
+| <a id="sig-specification"></a>Specification: General + OTel Maintainers Sync | Tuesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo) | [#otel-specification](https://cloud-native.slack.com/archives/C01N7PP1THC) | [calendar-spec-general](https://groups.google.com/a/opentelemetry.io/g/calendar-spec-general) |  |  |
```

Comment:
My text editor removed the whitespace at the end of the lines. If this is what's causing the table linter to fail, let me know and I can remove them.

Reply:
yeah, that's the issue

alternately, you can run `make table-generation`